test/fixtures/browserstack-app/app/src/main/java/com/bugsnag/android/mazerunner/MainActivity.kt

```diff
@@ -0,0 +1,34 @@
+package com.bugsnag.android.mazerunner
+
+import android.os.Build
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import android.widget.Button
+import com.bugsnag.android.*
```

Comment:
I prefer to avoid wildcard/star imports so it is obvious where the classes/interfaces come from.